migrating/ossm-migrating-read-me-assembly.adoc

```diff
@@ -0,0 +1,95 @@
+
+* link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-about-gateway-migration_gateway-migration[Gateway migration]
+
+include::modules/ossm-migrating-read-me-explicitly-create-openshift-routes.adoc[leveloffset=+1]
+include::modules/ossm-migrating-read-me-introducing-canary-upgrades.adoc[leveloffset=+1]
+
+.Next steps
+You can set `updateStrategy` to `RevisionBased` to use canary upgrades.
+
+* xref:../update/ossm-updating-openshift-service-mesh.adoc#about-revisionbased-strategy_ossm-performing-inplace-update[About RevisionBased strategy]
+
+Be aware that setting the `updateStrategy` to `RevisionBased` also has implications for some integrations with {SMProduct}, such as the cert-manager tool integration.
+
+* xref:../install/ossm-cert-manager-assembly.adoc#updating-istio-csr-revision-based-only_ossm-cert-manager-assembly[Updating istio-csr agents with revision-based update strategies]
+
+include::modules/ossm-migrating-read-me-supported-multi-cluster-topologies.adoc[leveloffset=+1]
+include::modules/ossm-migrating-read-me-support-for-istioctl.adoc[leveloffset=+1]
+include::modules/ossm-migrating-read-me-kubernetes-network-policy-management.adoc[leveloffset=+1]
+include::modules/ossm-migrating-read-me-tls-configuration-change.adoc[leveloffset=+1]
+
+//comments will be removed for OSSM 3.0 GA. For future reference only as Gwynne will likely be living in this file for Q1 2025.
+//Migrating dir will be similar to Observability dir structure. Something like:
+//Dir: migrating
+//Distros: openshift-service-mesh
+//Topics:
+//-Name: Migration Guides
+// File: ossm-file-name
+//- Name: Migrating Metrics
+//  Dir: mig dir name1
+//  Topics:
+//  - Name: Migrating metrics guide
+//    File: ossm-file-name2
+//- Name: Migratiing Traces
+//  Dir: mig dir name2
+// Topics:
+// - Name: Migrating traces
+//    File: ossm-file-name3
+//- Name: Migrating from Kiali to Kiali 2
+//  Dir: kiali
+//  Topics:
+//  - Name: Something about Kiali
+//    File: ossm-file-name4
+//Will be dependent on both the number of and type of migrations that need to take place to move from 2.6 to 3.
+//migrating-read-me is the only content we have as of 11/19/2024.
+//Possible file names will change once doc work can start on migration guides. As of 11/19/2024 it is unclear what migration guides will need to be created as engineering work is incomplete, and it is unknown how migration will happen. Will the operator have to go first? Do the integrations have to go in a particular order? Will there be differences if customers choose RevisionBased instead of the default InPlace update strategy? There are more questions than answers, so setting it up as best I can with the knowledge I have at this time.
+//Possible each module will have Next Steps that will link out to respective migration guide.
+
+[id="addition-resource_{context}"]
```

Comment:
Needs a role per the [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies):
```suggestion
[role="_additional-resources"]
[id="addition-resource_{context}"]
```